encoder.go

```diff
@@ -25,8 +25,8 @@ func ColoredLevelEncoder(level zapcore.Level, enc zapcore.PrimitiveArrayEncoder)
 
 // ColoredNameEncoder colorizes service names.
 func ColoredNameEncoder(s string, enc zapcore.PrimitiveArrayEncoder) {
-	if len(s) < 12 {
-		s += strings.Repeat(" ", 12-len(s))
+	if len(s) < 20 {
```

Comment:
Some custom plug-ins tend to have longer names due to their more personalized features